<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>HomeInfoBase</class>
  <widget class="QWidget" name="homeInfo" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>407</width>
        <height>315</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Form1</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item>
        <widget class="QTabWidget" name="TabWidget2" >
          <widget class="QWidget" name="tab" >
            <attribute name="title" >
              <string>&amp;Home info</string>
            </attribute>
            <layout class="QVBoxLayout" >
              <property name="margin" stdset="0" >
                <number>11</number>
              </property>
              <property name="spacing" stdset="0" >
                <number>6</number>
              </property>
              <item>
                <layout class="QGridLayout" >
                  <property name="margin" stdset="0" >
                    <number>0</number>
                  </property>
                  <property name="spacing" stdset="0" >
                    <number>6</number>
                  </property>
                  <item row="0" column="1" >
                    <widget class="Q3MultiLineEdit" name="edtAddress" />
                  </item>
                  <item row="2" column="0" >
                    <widget class="QLabel" name="TextLabel3" >
                      <property name="font" stdset="0" >
                        <font/>
                      </property>
                      <property name="text" stdset="0" >
                        <string>State:</string>
                      </property>
                      <property name="alignment" stdset="0" >
                        <set>Qt::AlignVCenter|Qt::AlignRight</set>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="0" >
                    <widget class="QLabel" name="TextLabel1" >
                      <property name="text" stdset="0" >
                        <string>Address:</string>
                      </property>
                      <property name="alignment" stdset="0" >
                        <set>Qt::AlignVCenter|Qt::AlignRight</set>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="2" column="1" >
                    <widget class="QLineEdit" name="edtState" />
                  </item>
                  <item row="3" column="1" >
                    <layout class="QHBoxLayout" >
                      <property name="margin" stdset="0" >
                        <number>0</number>
                      </property>
                      <property name="spacing" stdset="0" >
                        <number>6</number>
                      </property>
                      <item>
                        <widget class="QLineEdit" name="edtZip" />
                      </item>
                      <item>
                        <widget class="QLabel" name="TextLabel5" >
                          <property name="text" stdset="0" >
                            <string>Country:</string>
                          </property>
                          <property name="alignment" stdset="0" >
                            <set>Qt::AlignVCenter|Qt::AlignRight</set>
                          </property>
                          <property name="wordWrap" >
                            <bool>false</bool>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <widget class="QComboBox" name="cmbCountry" />
                      </item>
                      <item>
                        <widget class="QPushButton" name="btnWebLocation" >
                          <property name="sizePolicy" stdset="0" >
                            <sizepolicy>
                              <hsizetype>0</hsizetype>
                              <vsizetype>0</vsizetype>
                            </sizepolicy>
                          </property>
                          <property name="text" stdset="0" >
                            <string/>
                          </property>
                        </widget>
                      </item>
                      <item>
                        <spacer name="Spacer1" >
                          <property name="sizeHint" >
                            <size>
                              <width>20</width>
                              <height>20</height>
                            </size>
                          </property>
                          <property name="sizeType" >
                            <enum>QSizePolicy::Expanding</enum>
                          </property>
                          <property name="orientation" >
                            <enum>Qt::Horizontal</enum>
                          </property>
                        </spacer>
                      </item>
                    </layout>
                  </item>
                  <item row="1" column="0" >
                    <widget class="QLabel" name="TextLabel2" >
                      <property name="text" stdset="0" >
                        <string>City:</string>
                      </property>
                      <property name="alignment" stdset="0" >
                        <set>Qt::AlignVCenter|Qt::AlignRight</set>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="1" column="1" >
                    <widget class="QLineEdit" name="edtCity" />
                  </item>
                  <item row="3" column="0" >
                    <widget class="QLabel" name="TextLabel4" >
                      <property name="text" stdset="0" >
                        <string>Zip code:</string>
                      </property>
                      <property name="alignment" stdset="0" >
                        <set>Qt::AlignVCenter|Qt::AlignRight</set>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <widget class="Line" name="Line1" >
                  <property name="minimumSize" stdset="0" >
                    <size>
                      <width>0</width>
                      <height>5</height>
                    </size>
                  </property>
                  <property name="frameShape" stdset="0" >
                    <enum>QFrame::HLine</enum>
                  </property>
                  <property name="frameShadow" stdset="0" >
                    <enum>QFrame::Sunken</enum>
                  </property>
                  <property name="frameShape" stdset="0" >
                    <enum>QFrame::HLine</enum>
                  </property>
                </widget>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" stdset="0" >
                    <number>0</number>
                  </property>
                  <property name="spacing" stdset="0" >
                    <number>6</number>
                  </property>
                  <item>
                    <widget class="QLabel" name="TextLabel6" >
                      <property name="text" stdset="0" >
                        <string>Time zone:</string>
                      </property>
                      <property name="alignment" stdset="0" >
                        <set>Qt::AlignVCenter|Qt::AlignRight</set>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QComboBox" name="cmbZone" />
                  </item>
                  <item>
                    <spacer name="Spacer3" >
                      <property name="sizeHint" >
                        <size>
                          <width>20</width>
                          <height>20</height>
                        </size>
                      </property>
                      <property name="sizeType" >
                        <enum>QSizePolicy::Expanding</enum>
                      </property>
                      <property name="orientation" >
                        <enum>Qt::Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                </layout>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>QWidget</class>
      <extends>QWidget</extends>
      <header location="global" >QWidget</header>
      <sizehint>
        <width>-1</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
      <properties>
        <property type="String" >text</property>
      </properties>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>edtAddress</tabstop>
    <tabstop>edtCity</tabstop>
    <tabstop>edtState</tabstop>
    <tabstop>edtZip</tabstop>
    <tabstop>cmbCountry</tabstop>
    <tabstop>cmbZone</tabstop>
    <tabstop>TabWidget2</tabstop>
  </tabstops>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
</ui>
